frontend/src/components/WaiverPage.jsx

```diff
@@ -0,0 +1,158 @@
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import flower from "./HomePage/flower-bg.png";
+
+const Title1 = styled.h1`
+  color: white;
+  text-align: center
+  font-weight: 900;
+  font-size: 200%;
+  padding-bottom: 30px;
+`;
+
+const CenterWrap = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 1;
+`;
+
+const PlantContainer = styled.div`
+  position: absolute;
+  bottom: 0;
+  left: 5%;
+  z-index: 0;
+`;
+
+const BackGround = styled.div`
+  width: 80%;
+  height: 90%;
+  position: center;
+  background: #003c45;
+  border-radius: 80px;
+  padding: 20px;
+  border: center;
+  min-width: 800px;
+`;
+
+const FullPage = styled.div`
+  align-items: center;
+  background: #c9e8eb;
+  height: 100vh;
+`;
+
+const WaiverComp = styled.div`
+  width: 60%;
+  height: 70%;
+  background: #c4c4c4;
+  margin-left: 10%;
+`;
+
+const WaiverFormWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+`;
+
+const WaiverFormLeftWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: row;
+  flex-direction: column;
+`;
+
+const WaiverFormRightWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`;
+
+const AgreementSection = styled.div`
+  width: 100%;
+  height: 20%;
+  display: flex;
+  flex-direction: row;
+`;
+
+const AgreementText = styled.p`
+  color: white;
+  size: 200%;
+  margin-right: 10%;
+  font-size: 100%;
```

Comment:
Could you make the font-size a bit bigger? Also, its usually recommended to use pixels for font size, not percent